src/components/StepButtons.js

```diff
@@ -1,95 +1,55 @@
 import _ from 'lodash/fp'
 import { Fragment } from 'react'
-import { div, h, span } from 'react-hyperscript-helpers'
-import { Clickable } from 'src/components/common'
+import { div, h } from 'react-hyperscript-helpers'
+import Interactive from 'react-interactive'
 import { icon } from 'src/components/icons'
 import colors from 'src/libs/colors'
-
-
-export const params = {
-  buttonWidth: 170,
-  buttonHeight: 50,
-  dotSpace: 7,
-  dotSize: 6,
-  fontSize: 16
-}
+import * as Style from 'src/libs/style'
 
 
 const styles = {
   container: {
-    display: 'flex', alignItems: 'flex-start',
-    marginBottom: '2rem', marginTop: '1rem'
+    display: 'flex', alignItems: 'center',
+    fontWeight: 500, textTransform: 'uppercase',
+    marginTop: '1rem', height: '3.75rem'
   },
   button: isActive => ({
-    display: 'flex', alignItems: 'center',
-    flex: 'none',
-    width: params.buttonWidth, height: params.buttonHeight,
-    borderRadius: params.buttonHeight / 2, borderWidth: 2, borderStyle: 'solid',
-    borderColor: isActive ? colors.accent(1.2) : colors.dark(0.55),
-    backgroundColor: isActive ? colors.accent() : colors.dark(0.4),
-    color: 'white',
-    padding: '0 0.5rem 0 1.5rem'
+    ...Style.tabBar.tab,
+    ...(isActive ? Style.tabBar.active : {}),
+    backgroundColor: undefined,
+    fontSize: '1rem'
```

Comment:
nit: Is this the right font size? It looks a little bigger than what is in the mock, but it may just be an artifact of the mock